Zend/tests/inter_06.phpt

```diff
@@ -7,4 +7,4 @@ interface stdClass { }
 
 ?>
 --EXPECTF--
-Fatal error: Cannot declare interface stdClass, because the name is already in use in %s on line %d
+Fatal error: interface stdClass has already been declared in %s on line %d
```

Comment:
I'm wasn't sure if the `the name is already in use` part adds any useful context to the messages, so I got rid of it :)
With the new format, it is slighly annoying that the symbol type doesn't start with a capitalized letter.